### PR TITLE
test: some tests for depositTo

### PR DESCRIPTION
**Pull request by Chris-Hibbert**

### CODE REVIEW

File: packages/zoe/src/contractSupport/zoeHelpers.js [hunk 1]
```diff
@@ -319,7 +319,7 @@ export const depositToSeatSuccessMsg = `Deposit and reallocation successful.`;
  */
 
 export async function depositToSeat(zcf, recipientSeat, amounts, payments) {
-  assert(!recipientSeat.hasExited(), 'The recipientSeat cannot not be exited.');
+  assert(!recipientSeat.hasExited(), 'The recipientSeat must be active.');
```
Comment (katelynsills) [inline packages/zoe/src/contractSupport/zoeHelpers.js]: Hmm, we haven't used `active` as a term elsewhere. I don't like "cannot be exited" (whoops, there is a typo in this line) but it's obvious what we mean, even though it is awkward

**Review by katelynsills — APPROVED** [review]
Let's not call a seat `active` since we haven't used the term before, but other than that, looks great, thanks!

File: packages/zoe/src/contractSupport/zoeHelpers.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/src/contractSupport/zoeHelpers.js]: I realized we use `isLive` for payments. If we must use a positive term, can we use `live` instead?

Comment (Chris-Hibbert) [reply packages/zoe/src/contractSupport/zoeHelpers.js]: I changed it to "cannot have exited" for consistency with `hasExited()`.  It's not positive vs negative that I was reacting to, it's the agrammaticality of "can't be exited()".

**Pull request merged**